<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdvancedSettingsWidget</class>
 <widget class="QWidget" name="AdvancedSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>375</width>
    <height>371</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QCheckBox" name="downmixAudioCheckBox">
     <property name="text">
      <string>Downmix Audio from 5.1 to 2.1</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,1">
     <property name="bottomMargin">
      <number>8</number>
     </property>
     <item>
      <widget class="QCheckBox" name="doubleBufferCheckbox">
       <property name="text">
        <string>Double Buffering</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="doubleBufferWarningLabel">
       <property name="text">
        <string>Disable only if the client takes too much CPU</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="maximumLiveBufferLengthLabel">
     <property name="text">
      <string>Maximum Live Buffer Length</string>
     </property>
     <property name="buddy">
      <cstring>maximumLiveBufferLengthSlider</cstring>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="bottomMargin">
      <number>8</number>
     </property>
     <item>
      <widget class="QSpinBox" name="maximumLiveBufferLengthSpinBox">
       <property name="suffix">
        <string> ms</string>
       </property>
       <property name="minimum">
        <number>10</number>
       </property>
       <property name="maximum">
        <number>1000</number>
       </property>
       <property name="singleStep">
        <number>10</number>
       </property>
       <property name="value">
        <number>200</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="maximumLiveBufferLengthSlider">
       <property name="minimum">
        <number>10</number>
       </property>
       <property name="maximum">
        <number>1000</number>
       </property>
       <property name="singleStep">
        <number>10</number>
       </property>
       <property name="pageStep">
        <number>100</number>
       </property>
       <property name="value">
        <number>200</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>8</number>
     </property>
     <item>
      <widget class="QPushButton" name="browseLogsButton">
       <property name="text">
        <string>Browse Logs</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="clearCacheButton">
       <property name="text">
        <string>Clear Local Cache</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="resetAllWarningsButton">
       <property name="text">
        <string>Reset All Warnings</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>browseLogsButton</tabstop>
  <tabstop>clearCacheButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
